dashboard/checkit/wrapper/wrapper.sage

```diff
@@ -206,15 +206,15 @@ if len(sys.argv) >= 4:
     if sys.argv[3].lower() == "preview":
```

Comment:
here we ignore the amount if we're set to `"preview"`. So I think it's better to simply omit the `amount=10` earlier as it's misleading (it will use the default 1000 instead, but that's happily ignored here).

OR previews could support variable #s of seeds (maybe you really want to preview just 1 seed, or 100). But that's more work for minimal benefit?